core/domain/skill_services.py

```diff
@@ -747,6 +749,15 @@ def update_skill(committer_id, skill_id, change_list, commit_message):
     skill = apply_change_list(skill_id, change_list, committer_id)
     _save_skill(committer_id, skill, commit_message, change_list)
     create_skill_summary(skill.id)
+    misconception_is_deleted = any([
+        change.cmd == skill_domain.CMD_DELETE_SKILL_MISCONCEPTION
+        for change in change_list
+    ])
+    if misconception_is_deleted:
+        taskqueue_services.defer(
+            question_services.untag_deleted_misconceptions,
+            taskqueue_services.QUEUE_NAME_ONE_OFF_JOBS,
```

Comment:
As mentioned above I think passing the `deleted_misconception_ids` would be better here, wdyt?